lib/driver.js

```diff
@@ -248,17 +250,16 @@ class YouiEngineDriver extends BaseDriver {
       return res;
     }
 
+    const platformName = caps.platformName.toLowerCase();
+
     // make sure that the capabilities has youiEngineAppAddress
-    if (!caps.youiEngineAppAddress) {
+    if (!caps.youiEngineAppAddress && platformName !== 'noproxy') {
```

Comment:
There was a unit test for this case but the logic didn't match, have changed the logic to match but if it's the unit test itself that's incorrect please let me know 🙏